### PR TITLE
DDF-3934 Protect against class cast exception in PropertyJsonMetacardTransformer

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Adds a simple check to make sure we don't get a class cast exception when transforming metacards to json
#### Who is reviewing it? 
@emmberk @peterhuffer 

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
@andrewkfiedler
@rzwiefel

#### How should this be tested? (List steps with links to updated documentation)
CI build should be sufficient

#### What are the relevant tickets?
[DDF-3934](https://codice.atlassian.net/browse/DDF-3934)

### CODE REVIEW

**Comment by clockard** [comment]
build now

File: catalog/transformer/catalog-transformer-propertyjson-metacard/src/main/java/ddf/catalog/transformer/metacard/propertyjson/PropertyJsonMetacardTransformer.java [hunk 1]
```diff
@@ -181,6 +181,9 @@ private static Object convertValue(Serializable value, AttributeType.AttributeFo
 
     switch (format) {
       case DATE:
+        if (!(value instanceof Date)) {
+          return null;
```
Comment (rzwiefel) [inline catalog/transformer/catalog-transformer-propertyjson-metacard/src/main/java/ddf/catalog/transformer/metacard/propertyjson/PropertyJsonMetacardTransformer.java]: :question: Should we be logging something at debug or trace that we are dropping the date?

Comment (clockard) [reply catalog/transformer/catalog-transformer-propertyjson-metacard/src/main/java/ddf/catalog/transformer/metacard/propertyjson/PropertyJsonMetacardTransformer.java]: Wouldn't hurt. Will log a debug message here

**Comment by clockard** [comment]
build now

**Pull request merged**